src/containers/chat/chat-header/ChatHeader.tsx

```diff
@@ -76,6 +96,32 @@ const ChatHeader: FC<ChatHeaderProps> = ({
     setIsSearchOpen(false)
   }
 
+  const getAllMessages = useCallback(
+    () =>
+      messageService.getMessages({
+        chatId: currentChat._id,
+        limit: 100,
+        skip: 0
+      }),
+    [currentChat._id]
+  )
+
```

Comment:
why did you omit `setAllMessages` in the dependancy array?

Reply:
thanks for noticing)